### PR TITLE
Issues with classification of tags such as OpenAPI.

### PR DESCRIPTION
**Pull request by Modular-sys**
Issues with classification of tags such as OpenAPI.

### CODE REVIEW

File: src/main/java/com/power/doc/model/DocMapping.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package com.power.doc.model;
+
+import org.apache.commons.lang3.StringUtils;
+
```
Comment (shalousun) [inline src/main/java/com/power/doc/model/DocMapping.java:26]: The newly added code includes the license.

**Comment by shalousun** [comment]
For this PR, please add some detailed descriptions below:
1. What bug was fixed,
2. What new features were added for easier review. This will also make the purpose of the PR clear to the community.

**Comment by Modular-sys** [comment]
For the @tag annotations on classes and methods, which previously could not be categorized properly, the current handling method allows for unified management of tags. Each tag can correspond to both classes and methods, and each method can accurately find the tags it should possess. According to business relationships, a class contains multiple methods, and these methods should be able to locate the corresponding class. At the same time, they should inherit the class's tags.

File: src/main/java/com/power/doc/model/DocMapping.java (hunk 1 above)
Comment (Modular-sys) [reply src/main/java/com/power/doc/model/DocMapping.java:26]: MIT

Comment (shalousun) [reply src/main/java/com/power/doc/model/DocMapping.java:26]: You can refer to other smart-doc codes for this.

Comment (Modular-sys) [reply src/main/java/com/power/doc/model/DocMapping.java:26]: Added

Comment (shalousun) [reply src/main/java/com/power/doc/model/DocMapping.java:26]: license is thisimage

Comment (Modular-sys) [reply src/main/java/com/power/doc/model/DocMapping.java:26]: added

**Review by shalousun — APPROVED** [review]

**Pull request merged**